### DISCUSSION TITLE
Rival Parks

### DISCUSSION THREAD

**Discussion by crazyurbanmyth** [OpenRCT2/OpenRCT2#15784] [Feature requests]
Hello, I am new here just want to say what an awesome project I think this is. Ive looked around and I don't think anyone has already discussed this.
 My Idea is to have some sort of system of rival parks which could appear on charts to compare with your own park and possibly annually when the awards are given out to the parks show which park won what award (always used to annoy me when I had the tidiest park around and yet some how I did not get the award, I wanted to see a tidier park!!). 

Potentially if possible you could go see what these rival parks looked like even if they were always the same rival parks (ideally there would be a random pool of parks and say 5 - 10 would be selected as your rivals) and have them run by AI and see how they progress against your own park by having different charts such as park value, number of rides, number of guests, overall rating etc, or even having a set objective to earn more money than your rival parks. You could have different sets of rival parks including small/ begginers, normal/intimatadte and large/ expert. I understand that this would be very complex to implicate and this idea is for the distant future I just wanted to throw it out there to see what people think. Keep up the great work!

**Comment by Krutonium** [comment]
At the moment MP is possible, so what could happen could be a game mode like that, where each person has their park, and your competing.

Though for now, it probably won't be happening.

**Comment by Ryder17z** [comment]
That would be pretty cool

**Comment by ipatix** [comment]
It sounds fairly interesting and as much as I appreciate talking about that stuff, I think it's better to wait until the game has more or less finished to get implemented.

**Comment by Leftie76** [comment]
This'd be nice, far more worthwhile than co-operative play on the same thing.
I'd love to see two parks together, same size, same things available. First to reach X goal wins!
After X goal is reached, both players can continue to play just for fun if they'd like to.

**Comment by Dracheron** [comment]
This situation will come up now - how do you break ties if they finish in the same month (for scenarios with monthly income from a source) or if the objective is only tested once (for those where you have to get x number of guests by year y)?

**Comment by GeorgeWL** [comment]
> This situation will come up now - how do you break ties if they finish in the same month (for scenarios with monthly income from a source) or if the objective is only tested once (for those where you have to get x number of guests by year y)?

Possibly just don't? It's okay for people to have a draw.

Another possiblity though could be a "sudden death" where they get an extra week?

**Comment by James103** [comment]
Here's what I think how ties should be resolved when there are any in competitive play. Note that the criteria go from most relevant/important to the goal to less important but still relevant. Note that the criteria and ordering are **not finalized** and are subject to change.

| Scenario Objective          | How to Break Tie (1st criteria)         | 2nd                    | 3rd                 | 4th                 |
|-----------------------------|--------------------------------|------------------------|---------------------|---------------------|
| Number of guests            | Most guests in park            | Highest park rating    | Highest value       | First to reach goal* |
| Build 10 different coasters | Highest excitement             | Highest length         | Highest value       | First to reach goal* |
| Achieve monthly income      | Highest income of given source | Highest monthly income | Highest profit      | First to reach goal* |
| Achieve park value          | Highest value park             | Most guests in park    | Highest park rating | First to reach goal* |

\*If there is a tie there, then the game checks every day/week/month and evaluates the criteria in the table again (in order).

**Comment by AaronVanGeffen** [comment]
I would like to stress that this is not something that's _currently_ being worked on. As I said in #10138: 

> I agree, this would be a really interesting game mode. Once we have added support for maps, it might even be feasible. This is far from a trivial change, however.
> 
> Currently, all game data structures assume there is only one company. This would have to become more flexible before such a game mode can even be considered.

**Comment by crazyurbanmyth** [comment]
Nice to see this being brought up again even if it is still a long way off, I've been quietly playing and following this game progress and it is awesome to see, just thought I would stick my head out again to say great job people keep up the too work!
